File: sort_bubbles/sort_bubbles.js
function sortArray(array) {
  for (let j = 0; j < array.length - 1; j++) {
    for (let i = 0; i < array.length - 1; i++) {
      if (array[i] > array[i +1]) {
        let temp = array[i];
        array[i] = array[i + 1];
        array[i + 1] = temp;
      }
    }
  }

  return array;
}

File: sort_bubbles/sort_bubbles.rb
def sort_array(array)
  for j in 0...array.length
    for i in 0...array.length - 1
      current_element = array[i]
      next_element = array[i + 1]

      if current_element > next_element
        temp = current_element
        array[i] = next_element
        array[i + 1] = temp
      end
    end
  end

  array
end
